### DISCUSSION TITLE
Unable to stop/terminate/kill Ganache CLI?

### DISCUSSION THREAD

**Discussion by ArcusConsulting** [smartcontractkit/full-blockchain-solidity-course-py#208] [Q&A]
At 4:21, and I managed to successfully install Node.js, Yarn and Ganache CLI. But when I type 

ganache-cli

in my command line, there seems to be no way of exiting it. If I close my command line I get this
![ganache](https://user-images.githubusercontent.com/91853555/137572204-7dce80d6-f6da-4509-a49a-5cf9603aa4c3.jpg)
:

**Comment by ArcusConsulting** [comment]
I can't type in any commands below the "Listening on 127.0.0.1:8545". I tried pressing q or typing exit but nothing happens. I'm afraid to just close the terminal as it looks like this "process" is still running.

**Reply by ArcusConsulting** [reply]
I also don't want to open a second terminal and run ganache-cli [options] while my first terminal is still up.

**Reply by cromewar** [reply]
In order to stop running commands you should use `control + z` or `control + c` this will kill all the running process, give it a try ;)

**Reply by ArcusConsulting** [reply]
Yea i tried that and it doesn't work since i don't have another command prompt below the running process.

**Reply by cromewar** [reply]
it seems your are using git terminal or wsl2 terminal, right?
please go to the microsoft store app and install Windows Terminal, and give it a try using it ;)

![image](https://user-images.githubusercontent.com/20199180/137805706-a973298e-7169-4b47-9eb2-917d44c9212f.png)

**Comment by AlbertoSinigaglia** [comment]
`ganache-cli` is a "always running program", I mean that until you stop it, it will run (think it as a open program)... at that point, you can either choose to run it in background (`ganache-cli &`) or just launch it on another tab/window... for the closing warning, it's normal, windows is telling you that "there is something running", so interrupting it might cause problem (e.g. if you are copying some files)

**Reply by cromewar** [reply]
This is also true, ganache-cli is always running.
